docs/readme/README_examples.py

```diff
@@ -5,6 +5,7 @@
 import matplotlib.pyplot as plt
 import matplotlib.cm
 import neurokit2 as nk
+import PyEMD
```

Comment:
what's dis

Reply:
`signal_decompose` in readme need dis

Reply:
wait how come last time readme works without importing this